### PR TITLE
Cleanup setup

### PR DESCRIPTION
**Pull request by cmc333333**
Analogous to 18F/regulations-core#18

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
 from setuptools import setup, find_packages
 
 setup(
-    name = "regulations",
-    version = "0.1.0", 
-    license = "public domain", 
-    packages = find_packages()
+    name="regulations",
```
Comment (khandelwal) [inline setup.py]: Should the name have '-site' in it?

File: setup.py [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+    version="2.0.0", 
+    license="public domain",
```
Comment (khandelwal) [inline setup.py]: You might want to add CC0 as well, as public domain isn't a thing outside the US.

File: setup.py (hunk 1 above)
Comment (cmc333333) [inline setup.py]: The app has had the name `regulations` for a long time. I'm inclined to keep it that way for compatibility

File: setup.py (hunk 2 above)
Comment (cmc333333) [inline setup.py]: Sure

**Comment by cmc333333** [comment]
Updated with an explicit CC0 mention. Switched to a "Trove" classifier, per [recommendation](https://docs.python.org/2/distutils/setupscript.html#additional-meta-data)

**Pull request merged**